app/views/dashboard/dashboard.html.erb

```diff
@@ -27,54 +27,36 @@
 
   <div class="row">
 
-    <div class="col-lg-8">
+    <div class="col-lg-7">
 
       <%= render partial: "dashboard/header" %>
-
-      <div class="dashboard-blog">
-
-        <% if @blog %>
-
-          <% if @blog.main_image %>
-            <div class="img" style="background-image: url('<%= @blog.main_image.path(:large) %>')">
-          <% elsif @blog.scraped_image %>
-            <div class="img" style="background-image: url('<%= @blog.scraped_image %>')">
-          <% end %>
-
-          <div class="blog-caption">
-
-            <h4>
-              <small class="float-right d-none d-lg-inline"><%= raw translation('dashboard.dashboard.from_blog', :url => '/blog') %></small>
-              <a class="title" href="/blog"><%= @blog.title %></a>
-              <small><%= t('dashboard.dashboard.by') %>
-                <a href="/profile/<%= @blog.author.username %>"><%= @blog.author.username %></a></small>
-            </h4>
-
-          </div>
-
-          <% if @blog.main_image || @blog.scraped_image %>
-            </div>
+      <%= render partial: "dashboard/user_topics" %><br/>
+
+      <p style="width: 100%; text-align: center; border-bottom: 1px solid #ccc; line-height:2px; margin: 10px 0 20px; margin-left:-13px;">
+       <span style="background-color: #fff; padding: 0 10px; color: #808080;">Click to start following topics that interests you</span>
+      </p>
+      <br/>
+
+      <div>
+        <small><a href="/tags" class="form-grey" style="margin-left:-13px;">Trending topics:</a></small>
+        <span class="topics-span">
+         <% cache('trending-tags', expires_in: 24.hours, skip_digest: true) do %>
+            <% Tag.trending.each do |i| %>
```

Comment:
Maybe we can have better naming ? instead of `i`

Reply:
Sure!